lib/rules/value-no-vendor-prefix/README.md

```diff
@@ -55,12 +61,21 @@ a { background: linear-gradient(bottom, #000, #fff); }
 
 ## Optional secondary options
 
-### `ignoreValues: ["/regex/", /regex/, "string"]`
+### `ignoreValues`
+
+```json
+{ "ignoreValues": ["array", "of", "values", "/regex/"] }
+```
 
 Given:
 
 ```json
-["grab", "max-content", "/^-moz-all$/"]
+{
+  "value-no-vendor-prefix": [
+    true,
+    { "ignoreValues": ["grab", "max-content", "/^-moz-all$/"] }
```

Comment:
question

I suppose our code block linter detects invalid json?
e.g. `"ignoreValues": [/-linear-gradient$/, "max-content", "/^-moz-all$/"]`

Reply:
No. I previously used my tool [`remark-lint-code-block-syntax`](https://www.npmjs.com/package/remark-lint-code-block-syntax) in PR #7983, but we haven't installed it since the package is immature and has so many dependencies.

I locally tested all JSON codes with the package. No issues found.

Reply:
Then would it pose a problem to use `/regex/` examples for the code highlighter then?
I mean, if it doesn't I wouldn't mind keeping/adding them.

Reply:
Including a regex literal in JSON code would raise a few problems:
- Formatting via Prettier wouldn't work.
- Syntax highlighting would be broken. For example, a red background color is displayed for invalid JSON code on GitHub:
  ```json
  [/regex/]
  ```
- A copy-and-paste of invalid JSON code might break user JSON configs.

That's why I wouldn't like to have invalid JSON code. Instead, I believe the `"/regex/"` format description in `user-guide/configure.md` and its examples (e.g. `"/^-moz-all$/"`) are enough for now.